### PR TITLE
feat: updated component example look and feel

### PR DESCRIPTION
**Pull request by daylesalmon**
feat: Component examples now support displaying components in iframes
feat: Component examples can now show njk snippets and properties
feat: Component example can be scaled by dragging with mouse

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,8 +115,9 @@
     "webpack-stream": "^6.0.0"
   },
   "dependencies": {
-    "@babel/standalone": "^7.11.6",
     "@sentry/browser": "^5.22.3",
+    "gulp": "^4.0.2",
```
Comment (gldgrnt) [inline package.json:119]: Should this be a dev dependency?

**Review by catiarodriguescosta — CHANGES_REQUESTED** [review]
Great work here @daylesalmon. 
The things I could identify were components that appeared empty: 
- the Mobile app variant of Footer on [this page](https://deploy-preview-506--wmnetwork.netlify.app/patterns/header-and-footer/)
- the cookie banner on [this pahe](https://deploy-preview-506--wmnetwork.netlify.app/patterns/cookies/)

Also, indentation inside HTML looks weird, but I guess that is a non-issue for this PR.

File: debug.log [hunk 2]
```diff
@@ -0,0 +1 @@
+[1126/091032.579:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (gldgrnt) [inline debug.log:1]: This should probably be deleted / added to .gitignore

**Review by gldgrnt — CHANGES_REQUESTED** [review]
Great! Just a few incredibly minor things 👍🏾

File: package.json (hunk 1 above)
Comment (daylesalmon) [reply package.json:119]: Yup!

**Comment by daylesalmon** [comment]
> Great work here @daylesalmon.
> The things I could identify were components that appeared empty:
> 
> * the Mobile app variant of Footer on [this page](https://deploy-preview-506--wmnetwork.netlify.app/patterns/header-and-footer/)
> * the cookie banner on [this pahe](https://deploy-preview-506--wmnetwork.netlify.app/patterns/cookies/)
> 
> Also, indentation inside HTML looks weird, but I guess that is a non-issue for this PR.

I've fixed the mobile app variant...As the pattern is display:none on mobile devices and it is displayed in an iframe within the column it treated it as if it was mobile so wasn't showing anything (even though the HTML is there).
As a quick fix i've just took it out of the iframe so something shows.

The cookie issue...Looks as though this was already an issue on the live design system. I'll have a look and implement a fix.

**Review by gldgrnt — APPROVED** [review]
Looks really good to me! 👍🏾

**Review by catiarodriguescosta — APPROVED** [review]
Looks pretty good to me! 👍

**Pull request merged**